core/src/test/java/com/radixdlt/rev2/nofaucet/CallPreviewWithoutFaucetTest.java

```diff
@@ -0,0 +1,185 @@
+                new GenesisDataChunk.Stakes(
+                    ImmutableList.of(Address.virtualAccountAddress(validatorKey)),
+                    ImmutableList.of(
+                        tuple(
+                            validatorKey,
+                            ImmutableList.of(
+                                new GenesisStakeAllocation(
+                                    UInt32.fromNonNegativeInt(0), Decimal.ONE)))))),
+            Decimal.ofNonNegative(0L),
+            ImmutableList.of());
+
+    return DeterministicTest.builder()
+        .addPhysicalNodes(PhysicalNodeConfig.createBatch(1, true))
+        .messageSelector(firstSelector())
+        .messageMutator(MessageMutator.dropTimeouts())
+        .addModules(extraModules)
+        .functionalNodeModule(
+            new FunctionalRadixNodeModule(
+                FunctionalRadixNodeModule.NodeStorageConfig.tempFolder(folder),
+                true,
+                FunctionalRadixNodeModule.SafetyRecoveryConfig.BERKELEY_DB,
+                FunctionalRadixNodeModule.ConsensusConfig.of(1000),
+                FunctionalRadixNodeModule.LedgerConfig.stateComputerNoSync(
+                    StateComputerConfig.rev2().withGenesis(genesisData))));
+  }
+
+  @Test
+  public void call_preview_works_without_faucet() throws ApiException {
+    // Parameters corresponding to the following struct passed as input:
+    //    FungibleResourceManagerCreateWithInitialSupplyInput {
+    //      owner_role: Default::default(),
+    //      track_total_supply: Default::default(),
+    //      divisibility: Default::default(),
+    //      resource_roles: Default::default(),
+    //      metadata: Default::default()
+    //      address_reservation: Default::default(),
+    //      initial_supply: Default::default(),
+    //    }
+
+    final var addressing = Addressing.ofNetwork(Network.INTEGRATIONTESTNET);
+    final var accountLockerCall =
+        new TransactionCallPreviewRequest()
+            .network(Network.INTEGRATIONTESTNET.getLogicalName())
+            .target(
+                new BlueprintFunctionTargetIdentifier()
+                    .packageAddress(addressing.encode(ScryptoConstants.RESOURCE_PACKAGE_ADDRESS))
+                    .blueprintName("FungibleResourceManager")
+                    .functionName("create")
+                    .type(TargetIdentifierType.FUNCTION))
+            .addArgumentsItem("4d220000")
+            .addArgumentsItem("4d0100")
+            .addArgumentsItem("4d0700")
+            .addArgumentsItem("4d2106220000220000220000220000220000220000")
+            .addArgumentsItem("4d2102230c2100230c2200")
+            .addArgumentsItem("4d220000");
+
+    final var coreApiHelper = new ProtocolUpdateTestUtils.CoreApiHelper();
+    try (var test = createTest(coreApiHelper.module())) {
+      // Arrange: Start a single node network
+      test.startAllNodes();
+
+      // Act: Preview a transaction
+      final var callBeforeBottlenose =
```

Comment:
How do we know we're actually before bottlenose?

Reply:
I'd much rather we make it explicit / clear.

e.g. we can set a really long epoch in genesis so that anemone doesn't get enacted (for now) and add a comment explaining _why_ we have set the long epoch.

(And then, in the long-term, we should be able to generally reconfigure protocol update running in the test-runner, e.g. create a: `integration_test_no_scenarios_protocol_config() -> ProtocolConfig`, or use the `ProtocolUpdateContentOverrides` mechanism - or something, like we can now do on the rust side.

Reply:
Actually, this is also leftover from the test I've used as a base.